clusterloader2/testing/load/modules/measurements.yaml

```diff
@@ -0,0 +1,94 @@
+{{$USE_SIMPLE_LATENCY_QUERY := DefaultParam .USE_SIMPLE_LATENCY_QUERY false}}
+{{$ENABLE_NODE_LOCAL_DNS_LATENCY := DefaultParam .CL2_ENABLE_NODE_LOCAL_DNS_LATENCY false}}
+{{$NODE_LOCAL_DNS_LATENCY_THRESHOLD := DefaultParam .CL2_NODE_LOCAL_DNS_LATENCY_THRESHOLD "5s"}}
+
+steps:
+  - name: "{{$action}}ing measurements"
+    measurements:
+    {{if not $USE_SIMPLE_LATENCY_QUERY}}
+    - Identifier: APIResponsivenessPrometheus
+      Method: APIResponsivenessPrometheus
+      Params:
+        action: {{$action}}
+        enableViolations: {{$ENABLE_VIOLATIONS_FOR_API_CALL_PROMETHEUS}}
+        allowedSlowCalls: {{$ALLOWED_SLOW_API_CALLS}}
+        customThresholds: {{YamlQuote $CUSTOM_API_CALL_THRESHOLDS 4}}
+    {{end}}
+    - Identifier: APIResponsivenessPrometheusSimple
+      Method: APIResponsivenessPrometheus
+      Params:
+        action: {{$action}}
+        enableViolations: {{$ENABLE_VIOLATIONS_FOR_API_CALL_PROMETHEUS_SIMPLE}}
+        useSimpleLatencyQuery: true
+        summaryName: APIResponsivenessPrometheus_simple
+        allowedSlowCalls: {{$ALLOWED_SLOW_API_CALLS}}
+        customThresholds: {{YamlQuote $CUSTOM_API_CALL_THRESHOLDS 4}}
+    - Identifier: CreatePhasePodStartupLatency
+      Method: PodStartupLatency
+      Params:
+        action: {{$action}}
+        labelSelector: group = load
+        threshold: 1h # TODO(https://github.com/kubernetes/perf-tests/issues/1024): Ideally, this should be 5s
+    - Identifier: InClusterNetworkLatency
+      Method: InClusterNetworkLatency
+      Params:
+        action: {{$action}}
+        checkProbesReadyTimeout: {{$PROBE_MEASUREMENTS_CHECK_PROBES_READY_TIMEOUT}}
+        replicasPerProbe: {{AddInt 2 (DivideInt .Nodes 100)}}
+    {{if $ENABLE_NODE_LOCAL_DNS_LATENCY}}
+    - Identifier: NodeLocalDNSLatency
+      Method: NodeLocalDNSLatencyPrometheus
+      Params:
+        action: {{$action}}
+        enableViolations: true
+        threshold: {{$NODE_LOCAL_DNS_LATENCY_THRESHOLD}}
+    {{end}}
+    - Identifier: SLOMeasurement
+      Method: SLOMeasurement
+      Params:
+        action: {{$action}}
+        checkProbesReadyTimeout: {{$PROBE_MEASUREMENTS_CHECK_PROBES_READY_TIMEOUT}}
+        replicasPerProbe: {{AddInt 2 (DivideInt .Nodes 100)}}
+    {{if $PROMETHEUS_SCRAPE_KUBE_PROXY}}
+    - Identifier: NetworkProgrammingLatency
+      Method: NetworkProgrammingLatency
+      Params:
+        action: {{$action}}
+    {{end}}
+    {{if $ENABLE_API_AVAILABILITY_MEASUREMENT}}
+    - Identifier: APIAvailability
+      Method: APIAvailability
+      Params:
+        action: {{$action}}
+        pollFrequency: "5s"
+        hostPollTimeoutSeconds: 5
+    {{end}}
+    - Identifier: TestMetrics
+      Method: TestMetrics
+      Params:
+        action: {{$action}}
+        systemPodMetricsEnabled: {{$ENABLE_SYSTEM_POD_METRICS}}
+        clusterOOMsTrackerEnabled: {{$ENABLE_CLUSTER_OOMS_TRACKER}}
+        clusterOOMsIgnoredProcesses: {{$CLUSTER_OOMS_IGNORED_PROCESSES}}
```

Comment:
I'm getting 
`clusterOOMsIgnoredProcesses: null`
instead of
`clusterOOMsIgnoredProcesses: ""`

Could you look into that?